python/src/LeastSquaresEquationsSolver_doc.i.in

```diff
@@ -0,0 +1,47 @@
+%feature("docstring") OT::LeastSquaresEquationsSolver
+"Least Square Solver (LeastSquaresEquationsSolver) for a system of non-linear equations.
+
+
+Parameters
+----------
+absError : positive float
+    Absolute error: distance between two successive iterates at the end point.
+    Default is :math:`10^{-5}`.
```

Comment:
That is not true. The default value is from the `ResourceMap` key "Solver-DefaultAbsoluteError". Users should know this.